### ISSUE TITLE
MFDs beep, but no RPM screens

### ISSUE THREAD

**Issue by Blu3wolf** [icedown/KSP-ProbeControlRoom#5]
@linuxgurugamer, pinging you here as Issues are not available on your fork. Ive got this:
http://forum.kerbalspaceprogram.com/index.php?/topic/61250-104wipivaplugin-probecontrolroom-10-beta10a-iva-all-the-things-notsoshaky/&page=17#comment-2912693

> I hear the beep sound effect when I click buttons, but the screens never show anything, just fixed rpm-alike textures. 

> Having scansat installed makes the screens in the probe control room unresponsive.  

Thanks for maintaining the mod!

**Comment by icedown** [comment]
This is an issue with having so many RPM screens active at the same time.  The only thing I can really do is hack up the basic RPM screens and make it so that only specific pages are visible on certain monitors to limit the number of map pages, the ones that use scansat, get loaded.  

The mod itself doesn't add very much to the game, just a model and view point.  The functionality is all in the hands of RPM.  That being said, me and adding props are not getting along at the moment so rebuilding the individual RPM screens may be a good break.

**Comment by linuxgurugamer** [comment]
@Blu3wolf If you are referring to KVV, I just enabled the issues.  But it seems the issue is what @icedown said.
Does it work if you only have a few RPM screens active?

**Comment by Blu3wolf** [comment]
Ive gotten double the responses I expected! Okay. Icedown, I thought you had washed your hands of the mod. Umm. 
@Linuxgurugamer, none of the screens light up at all. So, I would say no RPM screens are active. Some of them beep when I push buttons. Others dont.

**Comment by icedown** [comment]
No, I just got tired of the crap on the forums and just decided to maintain it without a forum thread.  The map screen is very, very, hard on rpm processing time.  RPM limits the amount of time each screen gets per tick to keep speeds up, and with 17 rpm screens with that many different pages, it eats a lot of its time.  I'm working on thinning out the pages significantly, only allowing one two full mfds on each of the consoles, the other ones will all be stuck on a static page.

**Comment by linuxgurugamer** [comment]
@Blu3wolf Sounds like @icedown is on the right track.

**Comment by NexxusDrako313** [comment]
Some MFDs being stuck on a static page would be a good idea, so long as it were configurable. It should also save time when it comes to using new craft, which resets all the MFDs.

**Comment by icedown** [comment]
It's not something that can easily be left configurable since the layout of the pages has to be defined at startup.  when a vessel is loaded, all screens of all mfds are loaded.  Not all are functional but you can imagine the overhead of all the screens of 17 different mfds

**Comment by NexxusDrako313** [comment]
Ah, I didn't know that part (new to KSP modding), but yeah. I can imagine the overhead is quite... large.
